e2e/pages/adf/content-services.page.ts

```diff
@@ -356,12 +368,23 @@ export class ContentServicesPage {
         await this.createLibraryDialog.waitForDialogToOpen();
     }
 
-    async createNewFolder(folder): Promise<void> {
+    async createNewFolder(folder: string): Promise<void> {
```

Comment:
You can rename the folder to folderName just to keep similar declarations for all these functions

Reply:
done